### ISSUE TITLE
Project name: how do I search information about RELATE on internet? How do I refer to RELATE without awkwardness?

### ISSUE THREAD

**Issue by vi** [inducer/relate#191]
I try searching "RELATE" or "RELATE Django" and does not find this.

Initially I found this software by the name "CourseFlow". Searching for "CourseFlow" indeed proces the link to this repository as the first result.

How shall refer to RELATE when speaking about what is being deployed? If I say "RELATE", one may ask "relate to what?, it looks like a random part of sentence. 

Actually I like the name "CourseFlow" better.

**Comment by inducer** [comment]
Fair point. I liked CourseFlow, too! Unfortunately, at one point, there was a course system being sold at this URL:

http://www.alvaedu.com/courseflow/

under the exact same name, so I thought it wise to rename, to avoid a naming battle. Now, it seems like those people have gone away. And you're right "Relate" is surprisingly hard to google for, so it might not be the world's greatest name. I'll give it some thought.
